sdk/eventhub/azure-eventhub/samples/async_samples/sample_code_eventhub_async.py

```diff
@@ -207,8 +207,6 @@ async def on_event(partition_context, event):
     # [END eventhub_consumer_client_close_async]
 
 if __name__ == '__main__':
-    loop = asyncio.get_event_loop()
-    # loop.run_until_complete(example_eventhub_async_consumer_receive_and_close())
-    # loop.run_until_complete(example_eventhub_async_producer_send_and_close())
-    loop.run_until_complete(example_eventhub_async_send_and_receive())
-
+    # asyncio.run(example_eventhub_async_consumer_receive_and_close())
+    # asyncio.run(example_eventhub_async_producer_send_and_close())
```

Comment:
nit:
These may need to be uncommented? I think I may have been testing and accidentally committed them like this one time 🤦 this one's on me!

Reply:
good catch, uncommented